dev/com.ibm.ws.channelfw/src/com/ibm/ws/tcpchannel/internal/TCPChannelConfiguration.java

```diff
@@ -293,6 +295,15 @@ private void setValues() throws ChannelException {
                         continue;
                     }
 
+                    if (key.equalsIgnoreCase(START_OUTBOUND_SELECTORS_IMMEDIATELY)) {
```

Comment:
This statement is in a `isInbound()` block - I don't think that was your intention?